### DISCUSSION TITLE
any plans for android version?

### DISCUSSION THREAD

**Discussion by JunkyardCat** [flxzt/rnote#756] [Q&A]
Will there be plans for an android version

**Comment by Heiseneberg-maybe** [comment]
At #390 the idea and implementation is being tracked. 
At the moment there hasn't been any work done.
In the issue #245 the developer wrote "lets track progress for a general android app in https://github.com/flxzt/rnote/issues/390, even if it is unlikely to come ( as long as no other contributor spends some time and effort to create it )."

**Comment by Beryesa** [comment]
I wonder if there can be a file-format compatible side project for android.
I haven't taken a look at the source code yet but if the core logic and gtk bindings are modularly implemented, maybe a rust+flutter branch can be maintained without too much effort. (High likely much easier than trying to get any gtk on android?)

I don't have enough knowledge in either of them but in the case of no interest/time, perhaps I can try to step up and contribute, saving students from searching endlessly for a proper but cross platform foss annotation software, myself included, at some time in the future (next year?), together :3
